cardano_node_tests/tests/test_native_tokens.py

```diff
@@ -2067,6 +2067,44 @@ def test_long_name(
         exc_val = str(excinfo.value)
         assert "name exceeds 32 bytes" in exc_val or "expecting hexadecimal digit" in exc_val
 
+    @allure.link(helpers.get_vcs_link())
+    @hypothesis.given(token_amount=st.integers(min_value=MAX_TOKEN_AMOUNT + 1))
+    @common.hypothesis_settings()
+    def test_minting_amount_above_the_allowed(
+        self,
+        cluster: clusterlib.ClusterLib,
+        issuers_addrs: List[clusterlib.AddressRecord],
+        simple_script_policyid: Tuple[Path, str],
+        token_amount: int,
+    ):
+        """Test minting a token amount above the maximum allowed."""
+        temp_template = f"test_minting_amount_above_the_allowed_ci{cluster.cluster_id}"
+
+        asset_name_enc = temp_template.encode("utf-8").hex()
```

Comment:
Please add a random string to both `temp_template` and `asset_name_enc`. There are several intermediate files created during running the test and it is better to make sure there can't be a situation where we would use file created in some previous iteration of the test.

Reply:
Don't know if there is a better way to do it, if so, please tell me :)

Reply:
You can use
```
temp_template = common.get_test_id(cluster)
```
and there is a random string in the test ID.

Otherwise we have a helper function that we use like `clusterlib.get_rand_str(4)`